codegenerator/cli/npm/envio/src/Utils.res

```diff
@@ -154,7 +154,7 @@ Helper to check if a value exists in an array
     } else {
       let head = array->Js.Array2.slice(~start=0, ~end_=index)
       let tail = array->Belt.Array.sliceToEnd(index + 1)
-      [...head, ...tail]
+      Belt.Array.concat(head, tail)
```

Comment:
No reason other than my tree sitter plugin wasn't parsing spreads and annoying me 😅